src/foam/u2/Element.js

```diff
@@ -2257,10 +2257,12 @@ foam.CLASS({
 
           var propName = this.name.toLowerCase();
           var clsName  = this.forClass_.substring(this.forClass_.lastIndexOf('.') + 1).toLowerCase();
+          var canRead  = this.readPermissionRequired === false;
```

Comment:
Feel this should be !== true.   Thoughts?

Reply:
https://github.com/foam-framework/foam2/blob/master/src/foam/core/Property.js#L171-L175, I think it should be fine with === false.

Reply:
Property.js
`  {
      class: 'Boolean',
      name: 'readPermissionRequired',
      value: false
    },`